code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1330,19 +1331,23 @@
 	ADD_TRAIT(L, TRAIT_IGNOREDAMAGESLOWDOWN, type)
 	ADD_TRAIT(L, TRAIT_NOSTAMCRIT, type)
 	ADD_TRAIT(L, TRAIT_NOLIMBDISABLE, type)
-	ADD_TRAIT(L, TRAIT_NOBLOCK, type)
+	L.visible_message("<span class='warning'>You feel like nothing can stop you!</span>")
 
 /datum/reagent/stimulum/on_mob_end_metabolize(mob/living/L)
 	REMOVE_TRAIT(L, TRAIT_STUNIMMUNE, type)
 	REMOVE_TRAIT(L, TRAIT_SLEEPIMMUNE, type)
 	REMOVE_TRAIT(L, TRAIT_IGNOREDAMAGESLOWDOWN, type)
 	REMOVE_TRAIT(L, TRAIT_NOSTAMCRIT, type)
 	REMOVE_TRAIT(L, TRAIT_NOLIMBDISABLE, type)
-	REMOVE_TRAIT(L, TRAIT_NOBLOCK, type)
+	L.visible_message("<span class='warning'>You can feel your brief high wearing off</span>")
 	..()
 
 /datum/reagent/stimulum/on_mob_life(mob/living/carbon/M)
 	M.adjustStaminaLoss(-2*REM, 0)
+	M.losebreath += (current_cycle*0.05) // gradually builds up suffocation, will not be noticable for several ticks but effects will linger afterwards
```

Comment:
with only 15 cycles, you get `6 breathloss` and It looks too much in a short period. You might want `min(1, (current_cycle*0.05))` or something like this.

Reply:
I like and chose to use losebreath over direct oxyloss for one big reason - it allows the negative effects to easily persist after the reagent (and its beneficial effects) are gone. 

> with only 15 cycles, you get 6 breathloss and 20 cycles to 10.5 breathloss. It looks too much in a short period.

One breath gives you 5u of the reagent. 5u lasts for 25 cycles, or 50 seconds. I have balanced it with intent for it to be used one "huff" at a time like this, unless the user is willing and able to combine it with oxyloss medicines like perfluorodecalin.